non-standard-plugins.js

```diff
@@ -1,13 +1,13 @@
 module.exports = function shopifyNonStandardPlugins(options = {}) {
-  const {inlineEnv = false} = options;
+  const {inlineEnv = false, runtimePolyfill = false} = options;
```

Comment:
Do you think this should be on by default?

Reply:
Yes, but it would change builds that are leveraging runtime helpers. I'm curious how libraries are ensuring their consuming apps are using polyfills (`Object.assign`, `Array.from`, `Promise`, etc).

Reply:
I think libraries are generally just assuming those globals will be available, which I guess they are not when this option is on (since it seems to not do global polyfills?)